api/src/Repository/FiltersByCampCollaboration.php

```diff
@@ -15,26 +15,12 @@ trait FiltersByCampCollaboration {
      * the alias of the camp as the third argument if it's anything other than "camp".
      */
     public function filterByCampCollaboration(QueryBuilder $queryBuilder, User $user, string $campAlias = 'camp'): void {
-        $queryBuilder->leftJoin(
-            "{$campAlias}.collaborations",
-            "filter_{$campAlias}_campCollaboration",
-            Join::WITH,
-            $queryBuilder->expr()->andX(
-                $queryBuilder->expr()->eq("filter_{$campAlias}_campCollaboration.user", ':current_user'),
-                $queryBuilder->expr()->eq("filter_{$campAlias}_campCollaboration.status", ':established'),
-            )
-        );
-        $queryBuilder->andWhere(
-            $queryBuilder->expr()->orX(
-                // user is established collaborator in the camp
-                $queryBuilder->expr()->isNotNull("filter_{$campAlias}_campCollaboration.id"),
+        $campsQry = $queryBuilder->getEntityManager()->createQueryBuilder();
+        $campsQry->select('identity(uc.camp)');
```

Comment:
What is the purpose of `identity()`?

Reply:
Without `identity(..)` you would have to explicitly join the camp table so that the camp ID can be selected. 
With `Identity(..)` you get the ID of the referenced entity.

So you can save a join (to `Camp`).

See 
https://github.com/doctrine/orm/issues/6199